test/cases/ostree-common-functions.sh

```diff
@@ -0,0 +1,257 @@
+#!/bin/bash
+# A script that's full of common functions
+# used throughout our ostree tests. Can be 
+# sourced at beginning of those scripts.
+
+function CommonInit() {
```

Comment:
Thanks for extracting common helper, that is super nice!

(super nitpick) maybe uses common_init as the name as everything else in this file uses snake_case ?

Reply:
Thank you for the advice. I had it changed to snake_case.